opendevin/sandbox/e2b/sandbox.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, Tuple
+from e2b import Sandbox as E2BSandbox
+from e2b.sandbox.exception import (
+    TimeoutException,
+)
+
+from opendevin import config
+from opendevin.logger import opendevin_logger as logger
+from opendevin.sandbox.sandbox import Sandbox
+from opendevin.sandbox.e2b.process import E2BProcess
+from opendevin.sandbox.process import Process
+
+
+class E2Bbox(Sandbox):
+    closed = False
+    cur_background_id = 0
+    background_commands: Dict[int, Process] = {}
+
+    def __init__(
+        self,
+        template: str = 'base',
+        timeout: int = 120,
+    ):
+        self.sandbox = E2BSandbox(
+            api_key=config.get('E2B_API_KEY'),
```

Comment:
Just to confirm, since I haven't run this, is a good error thrown if `E2B_API_KEY` is not set? I feel that we should probably check that the key is set and throw a comprehensible error if not.

Reply:
We throw an exception if not API key is set - https://github.com/e2b-dev/E2B/blob/1221a57e7ec49d2e9fbc5ca1804d201bf5fa3673/packages/python-sdk/e2b/utils/api_key.py#L11

Would that be suffice?